docs/start/authentication/graphql-storefront.mdx

```diff
@@ -338,9 +338,9 @@ If your token is compromised, you can use the [Revoke a token](/docs/rest-authen
 A customer can only sign in on one device. When you run the customer login mutation for a session on a new device, the customer is automatically signed out of the previous session.
 </Callout>
 
-If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the login mutation to sign in to a customer account with an email address and a password. This will set a session cookie in the browser, which will authenticate the customer account on future requests.
+If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the login mutation to sign in to a customer account with an email address and a password. This will set a session cookie in the browser, which will authenticate the customer account on future requests. Optionally, you can use the login mutation to sign in using the cart ID of a guest cart associated with a customer ID. You must add the `guestCartEntityID` and its value as a GraphQL variable in mutations.
```

Comment:
`Optionally, you can use the login mutation to sign in using the cart ID` I think it is not the case .

`guestCartEntityID` is an argument which you can pass additionally to assign the cart to customer, which is performing log in operation.

Reply:
@bc-rafsierk thanks for looking at this. So I am not sure how to explain the changes added in [PAPI-2380](https://bigcommercecloud.atlassian.net/browse/PAPI-2380). If you do not add the cart ID in the login mutation. How do you assign it to a guest customer? Can you explain how this works?

Reply:
so basically, before the task was implemented the api user had to issue `login` mutation and after that to assign the cart to customer another mutation `assignCartToCustomer`. Thanks to PAPI-2380 this can be done in one `login` mutation by providing `guestCartEntityID `

Reply:
@bc-rafsierk  that helps. So I can remove the new text and replace it with the following. How does this sound?

You can also perform a login operation by passing the `guestCartEntityID` variable and assigning a cart to a customer.

Reply:
souds good 👍